File: news_manager_web/src/main/webapp/js/controller/userController.js
app.controller('userController',function ($scope,$controller, userService) {
    $controller('baseController',{$scope:$scope})

    $scope.init = function () {
        $scope.findPage();
    }

    $scope.findPage = function () {
        if(!$scope.keywords){
            $scope.keywords=''
        }
        userService.findPage($scope.paginationConf.currentPage,$scope.paginationConf.itemsPerPage,$scope.keywords).success(function (reponse) {
            $scope.list = reponse.rows;
            $scope.paginationConf.totalItems = reponse.total;
        });
    }


    $scope.findOne = function (id) {
        userService.findOne(id).success(function (response) {
            $scope.entity = response;
        });
    }
});

File: news_manager_web/src/main/java/cn/hncu/controller/UserController.java
package cn.hncu.controller;

import cn.hncu.entity.PageResult;
import cn.hncu.pojo.Userinfo;
import cn.hncu.service.IUserInfoService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * Created by Enzo Cotter on 2019/5/16.
 */
@RestController
@RequestMapping("/user")
public class UserController {

    @Autowired
    private IUserInfoService userService;

    @RequestMapping("/findPage")
    public PageResult findPage(int currentPage, int pageSize,String keywords){
        PageResult page = userService.findPage(currentPage, pageSize,keywords);
        return page;
    }

    @RequestMapping("/findOne")
    public Userinfo findOne(Integer id){
        return userService.findOne(id);
    }
}
